packages/ButtonGroup/__tests__/ButtonGroup.spec.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+
+import A11yContent from '@tds/core-a11y-content'
+
+import ButtonGroup from '../ButtonGroup'
+
+describe('ButtonGroup', () => {
+  const defaultProps = {
+    name: 'storageSize',
+    onChange: () => {
+      return true
+    },
+    value: '64gb',
+    label: 'Please select a storage size',
+  }
+  const doShallow = (props = {}) =>
+    shallow(
+      <ButtonGroup {...defaultProps} {...props}>
+        <ButtonGroup.Item value="64gb">64 GB</ButtonGroup.Item>
+        <ButtonGroup.Item value="128gb">128 GB</ButtonGroup.Item>
+        <ButtonGroup.Item value="256gb">256 GB</ButtonGroup.Item>
+      </ButtonGroup>
+    )
+
+  const doMount = (props = {}) =>
+    mount(
+      <ButtonGroup {...defaultProps} {...props}>
+        <ButtonGroup.Item value="64gb">64 GB</ButtonGroup.Item>
+        <ButtonGroup.Item value="128gb">128 GB</ButtonGroup.Item>
+        <ButtonGroup.Item value="256gb">256 GB</ButtonGroup.Item>
+      </ButtonGroup>
+    )
+
+  it('renders', () => {
+    const buttonGroup = doShallow(defaultProps)
```

Comment:
You don't need to pass in `defaultProps` to `doShallow` since it's already there.